web/src/features/App/api/postDeployAppVersion.tsx

```diff
@@ -75,6 +80,7 @@ function useDeployAppVersion({
       );
     },
     onSuccess: () => {
+      // TODO: figure out how to close the modal
       navigate(`/app/${slug}`);
```

Comment:
right now the modal stays open and it redirects to the newly deployed app page within the modal. need to figure out a way to close it since they don't share the same router/state.